Entities/Actor.cpp

```diff
@@ -1456,47 +1463,38 @@ void Actor::Update()
     /////////////////////////////////////////////
     // Take damage from large hits during travel
 
-    if (m_BodyHitSound && m_TravelImpulse.GetMagnitude() > m_TravelImpulseDamage / 2) {
-        m_BodyHitSound->Play(m_Pos);
-    }
+	float travelImpulseMagnitude = m_TravelImpulse.GetMagnitude();
+    if (travelImpulseMagnitude > m_TravelImpulseDamage * 0.5F) {
```

Comment:
Hmmm, I think this would be a bit cleaner as:
```cpp
	float travelImpulseMagnitude = m_TravelImpulse.GetMagnitude();

	if (m_BodyHitSound && travelImpulseMagnitude > m_TravelImpulseDamage * 0.5F) { m_BodyHitSound->Play(m_Pos); }

	if (travelImpulseMagnitude > m_TravelImpulseDamage) {
		const float impulse = travelImpulseMagnitude - m_TravelImpulseDamage;
		const float damage = std::max(impulse / (m_GibImpulseLimit - m_TravelImpulseDamage) * m_MaxHealth, 0.0F);
		m_Health -= damage;
		if (damage > 0 && m_Health > 0 && m_PainSound) { m_PainSound->Play(m_Pos); }
			if (m_Status != DYING && m_Status != DEAD) { m_Status = UNSTABLE; }
			m_ForceDeepCheck = true;
		}
	}
```
I'm not sure though, the sound playing and damage are related, so maybe they should be grouped after all. The nested if with a similar condition there just feels a bit weird to me haha.